CSCore.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+# Visual Studio 15
```

Comment:
Not sure whether this will break support for older versions of visual studio. Maybe we can switch it back? @ArizonaInkStudios